packages/shopify-codemod/transforms/simplify-?=-if-else-statement.js

```diff
@@ -0,0 +1,62 @@
+export default function simplifyCondAssignIfElseStatement({source}, {jscodeshift: j}, {printOptions = {quote: 'single'}}) {
+
+  function testIsBinaryExpression(path) {
+    return j.BinaryExpression.check(path.get('test').node);
+  }
+
+  function testChecksIdentifierForNotNull(path) {
+    const test = path.get('test').node;
+    return (
+      j.Identifier.check(test.left) &&
```

Comment:
I was under the assumption that decaf spat out the if statements from the Issue example: https://github.com/Shopify/javascript/issues/98#issue-156184628